cmd/kubeadm/app/cmd/alpha/certs.go

```diff
@@ -0,0 +1,167 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package alpha
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	kubeadmscheme "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/scheme"
+	kubeadmapiv1beta1 "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta1"
+	"k8s.io/kubernetes/cmd/kubeadm/app/cmd/options"
+	cmdutil "k8s.io/kubernetes/cmd/kubeadm/app/cmd/util"
+	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
+	certsphase "k8s.io/kubernetes/cmd/kubeadm/app/phases/certs"
+	"k8s.io/kubernetes/cmd/kubeadm/app/phases/certs/renewal"
```

Comment:
we need to move renewal outside of the phases namespace, IMHO.

Reply:
@neolit123 
This PR focuses on UX only, and it is already XXL, so I prefer to keep out of the field refactoring `/cmd/kubeadm/app/phases`.

Nevertheless I fully agree that we should clean up package structure, but IMO phases are a UX thing and should not be reflected outside `cmd/kubeadm/app/cmd`.
My idea is to rename `cmd/kubeadm/app/phases` into `cmd/kubeadm/app/pkg` (or in `pkg/kubeadm`, similar to other Kubernetes commnad), but this should be discussed at sig level before